app/src/main/java/com/vci/vectorcamapp/complete_session/details/presentation/components/specimens/CompleteSessionSpecimens.kt

```diff
@@ -52,9 +52,11 @@ fun CompleteSessionSpecimens(
                 modifier = Modifier.padding(
                     start = MaterialTheme.dimensions.spacingMedium,
                     end = MaterialTheme.dimensions.spacingMedium,
+                    top = MaterialTheme.dimensions.spacingSmall,
                     bottom = MaterialTheme.dimensions.spacingSmall
                 ),
-                onTooltipPrimaryAction = { onAction(CompleteSessionDetailsAction.ShowSearchTooltipDialog) },
+                isTooltipVisible = isSearchTooltipVisible,
```

Comment:
Question: Why did we add this isTooltipVisible variable in the previous commit? I see why we need it...but was it not present in the previous commit? How did you make it work initially without having this variable?